lascli/parser/workflows.py

```diff
@@ -85,9 +84,9 @@ def create_workflows_parser(subparsers):
     delete_workflow_parser.add_argument('workflow_id')
     delete_workflow_parser.set_defaults(cmd=delete_workflow)
 
-    delete_workflow_execution_parser = subparsers.add_parser('delete-execution')
-    delete_workflow_execution_parser.add_argument('workflow_id')
-    delete_workflow_execution_parser.add_argument('execution_id')
-    delete_workflow_execution_parser.set_defaults(cmd=delete_workflow_execution)
+    stop_workflow_execution_parser = subparsers.add_parser('stop-execution')
+    stop_workflow_execution_parser.add_argument('workflow_id')
+    stop_workflow_execution_parser.add_argument('execution_id')
+    stop_workflow_execution_parser.set_defaults(cmd=stop_workflow_execution)
```

Comment:
We are currently deleting the execution from the DB so I think this should be called 'delete'